blog/content/post/2020-06-15-qubes_summation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Qubes OS & 3mdeb 'minisummit' 2020 summation
+abstract: 'The second Qubes OS & 3mdeb minisummit is ahead of us. We had gone through four evenings of topics devoted to Qubes OS, so it is time for broad summation of the event.'
+cover: /covers/qubes&3mdeb_logo.png
+author: kamila.banecka
+layout: post
+published: true
+date: 2020-06-15
+archives: "2020"
+
+tags:
+  - QubesOS
+  - minisummit
+  - live
+  - YouTube
+  - firmware
+  - coreboot
+  - security
+  - TPM
+categories:
+  - Miscellaneous
+  - Security
+  - Firmware
+
+
+---
+The second Qubes OS and 3mdeb 'minisummit' is ahead of us. We have gone through
+four evenings of topics devoted to Qubes OS. It has been another fruitful and
+productive event, during which we have exchanged loads of knowledge and
+expertise. Below you will find list of topics, we have discussed with related
+videos and broad report of each day.
+
+
+* 20.05
```

Comment:
@KBanecka please also add slides for my presentations: https://cloud.3mdeb.com/index.php/f/247810
There are 3 PDFs there. For example:

```
  1. Michał Żygowski: Qubes OS on modern AMD platform
    * [recording](https://www.youtube.com/watch?v=Rw7rAPPyPPc&t=31s)
    * [slides](link)
```

@pietrushnic @Asiderr maybe consider publishing slides?

Reply:
https://www.slideshare.net/PiotrKrl/srtm-for-qubes-os-vms
https://www.slideshare.net/PiotrKrl/drtm-for-qubes-os-vms

Reply:
I am waiting for Norbert and Marek presentations